mypy/nodes.py

```diff
@@ -96,6 +96,16 @@ def get_column(self) -> int: pass
     'typing.Deque': '__mypy_collections__.deque',
 }
 
+reverse_collection_aliases = dict((name.replace('__mypy_collections__', 'collections'), alias)
+                                  for alias, name in
+                                  collections_type_aliases.items())  # type: Dict[str, str]
+
+nongen_builtins = {'builtins.tuple': 'typing.Tuple',
+                   'builtins.frozenset': 'typing.FrozenSet',
```

Comment:
Oh, it looks like an old oversight. I just checked stubs for ``builtins.frozenset`` and ``typing.FrozenSet``, the former is much better (it has all actual methods). I can easily fix this, but this requires a tiny PR to typeshed to be merged first and synced https://github.com/python/typeshed/pull/979

Could you please merge it?